src/components/Calculator.js

```diff
@@ -0,0 +1,26 @@
+const Calculator = () => (
+  <div className="mains">
+    <div className="output">0</div>
+    <div className="keys">AC</div>
+    <div className="keys">+/-</div>
+    <div className="keys">%</div>
+    <div className="sign">÷</div>
+    <div className="keys">7</div>
+    <div className="keys">8</div>
+    <div className="keys">9</div>
+    <div className="sign">*</div>
+    <div className="keys">4</div>
+    <div className="keys">5</div>
+    <div className="keys">6</div>
+    <div className="sign">-</div>
+    <div className="keys">1</div>
+    <div className="keys">2</div>
+    <div className="keys">3</div>
+    <div className="sign">+</div>
+    <div className="keys null">0</div>
+    <div className="keys">.</div>
+    <div className="sign">=</div>
+  </div>
+);
+
+export default Calculator;
```

Comment:
- [ ] @Habtamu210 as previously requested kindly add a child component for the Calculator component eg to display a line of your calculator, this makes your codebase neat and easier to scale in future

Reply:
@danmainah 
-Sorry can you brief me that I am not understanding it?

Reply:
in the project requirement, you were supposed to create a child component, a child component that has some functionality used in the main component, and data can be passed to the child component using props eg instead of writing all elements you could have a child component to display the output another displaying keys.

Reply:
in the project requirement you were suppose to create a child component, that has some functionality used in the main component and data can be passed to child component using props eg instead of writing all elements you could have a child component to display the output another displaying keys.